pkgs/development/python-modules/aeidon/default.nix

```diff
@@ -52,9 +42,10 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "aeidon" ];
 
   meta = with lib; {
+    changelog = "https://github.com/otsaloma/gaupol/releases/tag/${version}";
     description = "Reading, writing and manipulationg text-based subtitle files";
     homepage = "https://github.com/otsaloma/gaupol";
-    license = licenses.gpl3Only;
+    license = licenses.gpl3Plus;
```

Comment:
@erictapen Even the license was wrong. Please check all the others.

Reply:
Yeah I will